CMakeLists.txt

```diff
@@ -209,6 +209,13 @@ set_property(TARGET ${PROJECT_NAME} PROPERTY POSITION_INDEPENDENT_CODE ON)
 # INSTALL
 #################################################################################
 
+# libmd.pc for pkg-config
+set(DEST_DIR ${CMAKE_INSTALL_PREFIX})
+set(LIB -l${PROJECT_NAME}) # -lmd
+set(MD_VERSION ${MD_VERSION})
+configure_file(libmd.pc.in libmd.pc @ONLY)
+install (FILES libmd.pc DESTINATION /usr/lib64/pkgconfig)
```

Comment:
```
-install (FILES libmd.pc DESTINATION /usr/lib64/pkgconfig)
+install (FILES ${CMAKE_BINARY_DIR}/libmd.pc DESTINATION lib/pkgconfig)
```
If built out-of-source (e.g. `mkdir build && cd build && cmake -DCMAKE_INSTALL_PREFIX=$PWD/test -DMD_PLATFORM=linux -DMD_BUILD_TYPE=release .. && make && make install`), make install fails because it looks for libmd.pc at the source directory.

The `lib/pkgconfig` part is to respect the install prefix defined by the user.